mmv1/products/redis/Cluster.yaml

```diff
@@ -188,6 +188,20 @@ examples:
       'deletion_protection_enabled': 'false'
     oics_vars_overrides:
       'deletion_protection_enabled': 'false'
+  - name: "redis_cluster_cmek"
+    primary_resource_id: "cluster-cmek"
+    vars:
+      cluster_name: "cmek-cluster"
+      policy_name: "mypolicy"
+      subnet_name: "mysubnet"
+      network_name: "mynetwork"
+      kms_key_name: "mykey"
+      kms_ring_name: "mykeyring"
```

Comment:
Instead of creating a new KMS key, it would be better to use a bootstrapped key using BootstrapKMSKey (or one of the related functions): https://github.com/hashicorp/terraform-provider-google-beta/blob/689f8c4843ac18782984feb7c54749f7e9b394b3/google-beta/acctest/bootstrap_test_utils.go#L59 You can search the codebase to find examples of usage

Reply:
done